pytorch_memlab/extension.py

```diff
@@ -72,15 +84,15 @@ def mlrun(self, line=None, cell=None, local_ns=None):
         with profiler:
             exec(compile(code, filename='<ipython>', mode='exec'), local_ns)
 
-        if not args.quiet:
-            profiler.print_stats()
-
         if args.dump_profile is not None:
             with open(args.dump_profile, 'w') as f:
                 profiler.print_stats(stream=f)
 
         if args.return_profiler:
             return profiler
+        else:
+            defaults = [] if args.no_default_columns else DEFAULT_COLUMNS
+            return profiler.display(columns=defaults + args.column)
```

Comment:
Most of the time you run the profiler, you're interested in peak allocation. Specifying `-c active_bytes.all.peak` every time is a pain. Problem is that if I put it in the default argument list for `column`, then - because it uses an append action - you wouldn't be able to override them the rare time that you don't actually want those columns at all.